src/resmom/mom_main.c

```diff
@@ -7044,6 +6897,18 @@ encode_used(job *pjob, pbs_list_head *phead)
 	}
 }
 
+/**
+ * @brief
+ * 	check give attribute name exists in give ruu's attr list
+ *
+ * @param[in] old  - pointer to ruu
+ * @param[in] name - attribute name
+ *
+ * @return int
+ * @retval 0 - doesn't exists in ruu's attr list
+ * @retval 1 - exists in ruu's attr list
+ *
+ */
 static int
 is_in_old_ruu(ruu *old, char *name)
```

Comment:
as discussed offline, let us remove this function and use the logic in the caller (just one).